compiler/pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+filterwarnings =
+    default
+    ; Warning in cookies module triggered by responses lib.
+    ignore:::cookies
+    ; Pandas triggers a bug in the current version of Cython.
+    ; Unfortunately, we lose monitoring of any potential ImportWarnings,
+    ignore::ImportWarning
+
```

Comment:
Was getting extraneous warnings from pytest.  The 'ignore::ImportWarning' is broad, but can be removed once cython bugfix is released.